proto/streamlit/proto/Audio.proto

```diff
@@ -26,4 +26,7 @@ message Audio {
 
   // The currentTime attribute of the HTML <audio> tag's <source> subtag.
   int32 start_time = 3;
+
+  // A url pointing to an audio file
+  string url = 4;
```

Comment:
Since you can only have either `data` or `url`, move these into a `oneof`:

```
oneof src {
  // The data in Base64 format.
  string data = 1;

  // A url pointing to an audio file
  string url = 4;
}
```

More info: https://developers.google.com/protocol-buffers/docs/proto3#oneof